lib/open_pantry/blank.ex

```diff
@@ -4,25 +4,57 @@ defmodule Blank do
   """
 
   @doc """
-  Returns `true` if data is considered blank/empty.
+  Returns `true` if data is considered blank/empty.  What is considered blank?
+
+  * Empty strings, empty lists, and empty maps
+  * The atoms `false` and `nil`
+
+  Everything else is non-blank.  That includes whitespace-only strings, and all integers, even 0.  (Other types of numbers have not been implemented yet.)
+
+  ## Examples
```

Comment:
30 lines of doc tests for this simple a function still feels like overkill to me.. I guess I can merge in and fix myself later if you want, but still think 2 or 3 good examples is plenty here and don't want to set a confusing precedent for use of doc tests elsewhere in code that this is how we do them.

Also if fixing this up wouldn't mind adding a link to the gist I linked above to provide attribution.